Models/Customers.cs

```diff
@@ -0,0 +1,33 @@
+﻿
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace BangazonAPI.Models
+{
+    public class Customers
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [Required]
+        public string FirstName { get; set; }
+        [Required]
+        public string LastName { get; set; }
+
+        [Required]
+        [DataType(DataType.Date)]
+        [DatabaseGenerated(DatabaseGeneratedOption.Computed)]
+        public DateTime DateCreated { get; set; }
+
+        [Required]
+        [DataType(DataType.Date)]
+        [DatabaseGenerated(DatabaseGeneratedOption.Computed)]
+        public DateTime LastActivity { get; set; }
+
+        List<Customers> CustomerList = new List<Customers>();
```

Comment:
A customer won't actually have a list of customers inside it.
A customer DOES need two lists-- check the customers ticket -- you will need lists for the things that can be included
https://github.com/nashville-software-school/csharp-bangazonapi-boilerplate/issues/1